mri_watershed/brain_volume/mri_brain_volume.cpp

```diff
@@ -1044,7 +1047,14 @@ static int Pre_CharSorting(STRIP_PARMS *parms,MRI_variables *MRI_var) {
   for (k=0;k<r;k++)
     for (j=0;j<r;j++)
       for (i=0;i<r;i++) {
+#if GCC_VERSION > 80000
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wint-in-bool-context"
+#endif
         if (!(i*j*k*(i-r+1)*(j-r+1)*(k-r+1))) {
+#if GCC_VERSION > 80000
+#pragma GCC diagnostic pop
+#endif
```

Comment:
note that the error for this (and `mri_watershed`) said:
```
/code/mri_watershed/brain_volume/mri_brain_volume.cpp:1051:36: error: '*' in boolean context, suggest '&&' instead [-Werror=int-in-bool-context]
```
(pointing to the last `*` in line 1054). I'm not sure what that means/what's happening right here so I ignored the warning)

Reply:
My guess is that something like `((i*j*k*(i-r+1)*(j-r+1)*(k-r+1)) == 0)` would silence this

Reply:
ok i made this this change in the 2 places where it occurs